Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -15,6 +15,7 @@
 import SwiftUI
 import ArcGIS
 
+@MainActor
 public struct PreplannedListItemView: View {
```

Comment:
This also needs the preconcurrency attribute since it is public. The MainActor annotation adds a new constraint to this view and since we don't know how everyone uses this type—they may have extended it with their own custom code, for example—we need the preconcurrency warning to prevent potentially breaking their source.

```suggestion
@MainActor
@preconcurrency
public struct PreplannedListItemView: View {
```